deploy/bz1883589-kubeapiserver-revert/00-cluster.kubeapiserver.patch.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: operator.openshift.io/v1
+kind: KubeAPIServer
+name: cluster
+applyMode: AlwaysApply
+# https://bugzilla.redhat.com/show_bug.cgi?id=1883589#c8
+patch: '[{ "op": "remove", "path": "/spec/unsupportedConfigOverrides" }]'
+patchType: json
```

Comment:
If we have any other overrides set this would remove all of them. Can we remove just `/spec/unsupportedConfigOverrides/apiServerArguments` ?

Reply:
@jewzaam any other config overrides would block upgrades. I believe we have to remove the entire section if we want 4.6 upgrades to be unblocked

Reply:
If we create additional overrides they may not be fixed in the same z as this one.  It's a risk, small and not likely.